docs/Air.md

```diff
@@ -238,6 +239,21 @@ If the PNR contains no active segments it could not be imported into uAPI. Thus
 
 **See: <a href="../examples/Air/getUniversalRecordByPNR.js">Example</a>**
 
+## .getUniversalRecord(params)
+<a name="getUniversalRecord"></a>
+> May require Terminal access enabled in uAPI. See [TerminalService](Terminal.md)
```

Comment:
It seems that this method does not use terminal service

Reply:
My bad, I just copied from above.